src/include/duckdb/parser/parsed_data/create_table_info.hpp

```diff
@@ -30,6 +31,8 @@ struct CreateTableInfo : public CreateInfo {
 	vector<unique_ptr<Constraint>> constraints;
 	//! CREATE TABLE from QUERY
 	unique_ptr<SelectStatement> query;
+	//! Column dependency manager, used in CREATE TABLE to add dependencies before the table is constructed
+	ColumnDependencyManager column_dependency_manager;
```

Comment:
Probably should be moved to the `BoundCreateTableInfo` and generated during the binding, rather than during the transforming